00-HTML-CSS-basics/ex1.VI.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Ex 1.VI</title>
+		<meta charset="UTF-8">
+		<meta name="author" content="Martin Kalaizich">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+<form>
+First name:<br><input type="text" name="firstname" required><br>
```

Comment:
You can just wrap this labels inside a block element or even a `<label>` tag. As well as you can do for `inputs`. If you do that you can get rid of those unnecesary `<br>`